biomage/experiment/download.py

```diff
@@ -490,44 +462,61 @@ def download(
 
         elif file == RAW_FILE:
             print("\n== Downloading raw RDS file")
-            _download_raw_rds_files(
-                experiment_id,
-                input_env,
-                output_path,
-                name_with_id,
-                without_tunnel,
-                boto3_session,
-                aws_account_id,
-                aws_profile,
-            )
+            try:
```

Comment:
what is this client error handling that you added here?

Reply:
Hmm yea, maybe better to just catch all errors

Reply:
I initially put only client errors because I was handling for 404 if the file is not found. Was thinking if something else happens, there might be other cause of errors